src/Sulu/Bundle/CategoryBundle/Resources/public/js/components/categories/edit/details/main.js

```diff
@@ -172,6 +172,7 @@ define(['jquery', 'services/sulucategory/category-manager', 'text!./form.html'],
                 {
                     el: this.$find('#keywords-list'),
                     url: this.templates.keywordsUrl({category: this.data.id, locale: this.options.locale}),
+                    storageName: 'category-' + this.data.id + 'keywords',
```

Comment:
I know that it doesn't change the behavior, but it would be a BC break if it drives us crazy one day, and we change it 😉